workflow/executor/docker/docker.go

```diff
@@ -108,7 +108,10 @@ func (d *DockerExecutor) Kill(containerIDs []string) error {
 	killArgs := append([]string{"kill", "--signal", "TERM"}, containerIDs...)
 	// docker kill will return with an error if a container has terminated already, which is not an error in this case.
 	// We therefore ignore any error. docker wait that follows will re-raise any other error with the container.
-	_ := common.RunCommand("docker", killArgs...)
+	err := common.RunCommand("docker", killArgs...)
+	if err != nil {
+		log.Infof("docker kill --signal TERM: %s", err)
```

Comment:
warning? maybe state you're ignoring it in the message?

Reply:
ok, changed to warning that error is ignored.